### PR TITLE
[KOGITO-1884] - Clean up old cucumber test namespaces

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/KOGITO-1884

Signed-off-by: Mauricio Magnani <mmagnani@redhat.com>

### CODE REVIEW

File: tools/cleanup-namespace/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+date_cleanup=$(date -d "1 day ago" +%s)
+
+for values in $projects
+do
+    name_project=${values%;*}
+    date_project=${values#*;}
+
+    if [[ "$date_project" < "$date_cleanup"  ]] ; then
+        oc delete project $name_project
```
Comment (radtriste) [inline tools/cleanup-namespace/cleanup.sh:14]: can we add a small `echo` to display which project is deleted ?

**Review by radtriste — COMMENTED** [review]
Tested briefly and did not work...

here where the displayed entries:
```
date_cleanup = 1589105345
name_project = tradisso-test
date_project = 2020-05-04T08:58:51Z
```

Obviously `date_project` is not in the same format as `date_cleanup`.

```
$ kubectl get namespace tradisso-test -o yaml
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    openshift.io/description: ""
    openshift.io/display-name: ""
  creationTimestamp: "2020-05-04T08:58:51Z"
  name: tradisso-test
  resourceVersion: "15180669"
  selfLink: /api/v1/namespaces/tradisso-test
  uid: 8a860831-a8a2-4228-be44-9d2a36e937c9
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
```

File: tools/cleanup-namespace/cleanup.sh (hunk 1 above)
Comment (msmagnanijr) [reply tools/cleanup-namespace/cleanup.sh:14]: @radtriste sure, i will add the echo. I'm sorry, I should have left this as a draft, I'm still testing it.

Comment (spolti) [reply tools/cleanup-namespace/cleanup.sh:14]: @msmagnanijr let us know when you finish your changes.
you can tag this jira as draft as well.
thanks.

**Comment by spolti** [comment]
@msmagnanijr do you have updates on this?

**Comment by radtriste** [comment]
I guess you could also create a Jenkinsfile that will call this script and that we can then use in a Jenkins job.
wdyt ?

**Pull request closed without merging**